viewer/packages/camera-manager/src/StationaryCameraManager.ts

```diff
@@ -138,6 +144,11 @@ export class StationaryCameraManager implements CameraManager {
     });
   }
 
+  setFOV(fov: number): void {
+    this._camera.fov = THREE.MathUtils.clamp(fov, this._minFOV, this._defaultFOV);
+    this._cameraChangedListeners.forEach(cb => cb(this._camera.position, this._camera.position));
```

Comment:
Is it correct with `this._camera.position, this._camera.position` here? :monocle_face: I see it's done similarly further down in the file, but seems odd

Reply:
Added getTarget() 👍